src/coreclr/jit/fgbasic.cpp

```diff
@@ -2084,6 +2084,19 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, Fixed
                 goto OBSERVE_OPCODE;
             }
 
+            case CEE_NOCHECK:
+            {
+                noway_assert(sz == 0);
+
+                int flags = getU1LittleEndian(codeAddr);
+
+                prefixFlags |= (flags <<= 6);
```

Comment:
Can we just have the three flag checks and avoid having this cross-enum dependency?